### PR TITLE
Fetch stat var group info only from frequent import group

### PR DESCRIPTION
**Pull request by shifucun**
I updated the import group version to use the latest frequent group to verify the golden response is as intended.

### CODE REVIEW

File: internal/server/statvar/golden/get_statvar_group/earth.json [hunk 1]
```diff
@@ -209,9 +209,9 @@
         },
         {
           "id": "Count_Person_IsInternetUser_PerCapita",
-          "searchName": "Percent of Internet Users",
+          "searchName": "Population Count of Person With is Internet User is True (As Fraction of Per Capita)",
```
Comment (shifucun) [inline internal/server/statvar/golden/get_statvar_group/earth.json:212]: @pradh Is this expected from recent hierarchy name change?

File: internal/server/statvar/golden/get_statvar_group_node/root.json [hunk 2]
```diff
@@ -53,13 +53,13 @@
       "id": "dc/g/Crime",
       "specializedEntity": "Crime",
       "displayName": "Crime",
-      "descendentStatVarCount": 12308
+      "descendentStatVarCount": 8528
```
Comment (shifucun) [inline internal/server/statvar/golden/get_statvar_group_node/root.json:56]: Crime stat vars dropped substantially from recent hate crime schema update.

File: internal/store/bigtable/group.go [hunk 3]
```diff
@@ -74,6 +74,18 @@ func NewGroup(
 	}
 }
 
+// GetFrequentGroup creates a group that only has frequent import group table.
+func GetFrequentGroup(g *Group) *Group {
+	result := &Group{tables: []*Table{}}
+	for _, t := range g.tables {
+		if strings.HasPrefix(t.name, "frequent") {
```
Comment (pradh) [inline internal/store/bigtable/group.go:81]: "frequent_" for safety?

File: internal/server/statvar/statvar_group.go [hunk 4]
```diff
@@ -137,12 +137,12 @@ func GetStatVarGroup(
...
-		// Read stat var group cache from the first table, which is the most
-		// preferred cache BigTable. Since stat var schemas are rebuild with every
-		// group, so no merge needed.
+		// Read stat var group cache from the frequent import group table. It has
+		// the latest and trustworthy stat var schemas and no need to merge with
+		// other import groups.
 		btDataList, err := bigtable.Read(
 			ctx,
-			store.BtGroup,
+			bigtable.GetFrequentGroup(store.BtGroup),
```
Comment (pradh) [inline internal/server/statvar/statvar_group.go:145]: Just to check, are there other cache types where we can read from the best one (whichever it is) without reading everything and picking the first?

**Review by pradh — APPROVED** [review]
Nice!

File: internal/server/statvar/golden/get_statvar_group/earth.json [hunk 5]
```diff
@@ -209,9 +209,9 @@
...
           "searchNames": [
```
[the remark this answers is not in the document]
Comment (shifucun) [reply internal/server/statvar/golden/get_statvar_group/earth.json:213]: Yes, can drop searchName now.

File: internal/server/statvar/statvar_group.go (hunk 4 above)
Comment (shifucun) [reply internal/server/statvar/statvar_group.go:145]: Seems only the stat var hierarchy is standalone; Other caches all span schema and data/stat..

File: internal/store/bigtable/group.go (hunk 3 above)
Comment (shifucun) [reply internal/store/bigtable/group.go]: Done

**Pull request merged**